app/assets/stylesheets/landing.css.scss

```diff
@@ -0,0 +1,5 @@
+
+#main-logo {
+	height: 100%;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered background-color, height